src/components/PlayerCard/PlayerCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import { Props } from './types';
+
+export default function PlayerCard(props: Props) {
+  const [readyStatus, setReadyStatus] = useState(false);
+  useEffect(() => {
+    if (props.playerType === 'bot') {
+      setReadyStatus(true);
+    }
+  }, [props.playerType]);
+  return (
```

Comment:
🧐
Это вопросы код-стайла и, возможно, вы решили писать иначе, но кажется, что так читабельнее:
```suggestion
  const [readyStatus, setReadyStatus] = useState(false);
  
  useEffect(() => {
    if (props.playerType === 'bot') {
      setReadyStatus(true);
    }
  }, [props.playerType]);
  
  return (
```

Блоки кода, в частности блоки объявления переменных и `return`-ы, хорошо бы отбивать пустой строкой

Reply:
Да, так выглядит лучше, но Prettier после форматирования убирает пробелы между строками.
Подскажи, пожалуйста, есть ли настройка чтобы не исправлять такие моменты вручную после автоматического форматирования?

Reply:
ни eslint, ни prettier не убирает пустые строки между блоками кода. Оставляют одну, если их 2 или больше подряд, и добавляют в конце файла пустую строку. 
Может это ide так форматирует, как ты запускаешь автоматическое форматирование? через `npm run lint:fix`?

Reply:
Хороший вопрос, почему-то сейчас не смог повторить проблему при автоматическом форматировании, когда автоматически удалялись примеры. Возможно просто баг был.